pkg/resolve/resolve_test.go

```diff
@@ -34,6 +34,7 @@ func setup() {
 
 // Not sure how to get testParams fixtures working
 func readTDfile(t *testing.T, testname string, generateName bool, remoteTasking bool) (*tektonv1beta1.PipelineRun, *zapobserver.ObservedLogs, error) {
+	t.Helper()
```

Comment:
there is a bunch of them everywhere we didn't set as helper, we should update it.

Reply:
yeah.. I shouldn't have commited this one here, and instead create a PR just for it but 😐

Reply:
no worries...